public/tas.html

```diff
@@ -51,47 +60,52 @@
-              <h3>2.  Send our letter to these Representatives</h3>
-              <div class="pics-container">
-                <section class="pol-pics" id="obama">
-                  <img src="photos/obama.jpg" alt="http://a5.files.biography.com/image/upload/c_fill,cs_srgb,dpr_1.0,g_face,h_300,q_80,w_300/MTE4MDAzNDEwNzg5ODI4MTEw.jpg" title="Barack Obama" class="pol-button" id="president@whitehouse.gov">
-                  <p>President Barack Obama</p>
-                </section>
-                <section class="pol-pics" id="inslee">
-                  <img src="photos/Jay_Inslee.jpg" alt="http://upload.wikimedia.org/wikipedia/commons/thumb/6/6e/Jay_Inslee,_Official_Portrait,_c112th_Congress.jpg/220px-Jay_Inslee,_Official_Portrait,_c112th_Congress.jpg" title="Governor Inslee" class="pol-button" id="Jay.Inslee@gov.wa.gov">
-                  <p>Govenor Jay Inslee</p>
-                </section>
-                <section class="pol-pics" id="cantwell">
-                  <img src="photos/Maria_Cantwell.jpg" alt="http://upload.wikimedia.org/wikipedia/commons/thumb/0/02/Maria_Cantwell,_official_portrait,_110th_Congress.jpg/220px-Maria_Cantwell,_official_portrait,_110th_Congress.jpg" title="Senator Cantwell" class="pol-button" id="maria_cantwell@cantwell.senate.gov">
-                  <p>Senator Maria Cantwell</p>
-                </section>
-                <section class="pol-pics" id="murray">
-                  <img src="photos/Patty_Murray.jpg" alt="http://upload.wikimedia.org/wikipedia/commons/4/4f/Patty_Murray,_official_portrait,_113th_Congress.jpg" title="Senator Murray" class="pol-button" id="Senator@murray.senate.gov">
-                  <p>Senator Patty Murray</p>
-                </section>
-                <section class="pol-pics" id="mcdermott">
-                  <img src="photos/Jim_McDermott.jpg" alt="https://www.congress.gov/img/member/112_mcdermott_wa07.jpg" title="Congressman McDermott" class="pol-button" id="ima_wa07iq@mail.house.gov">
-                  <p>Congressman Jim McDermott</p>
-                </section>
-                <section class="pol-pics" id="jewell">
-                  <img src="photos/Sally_Jewell.jpg" alt="http://www.dpnm.net/wp-content/uploads/2014/01/jewell.jpg" title="Sec. Jewell" class="pol-button" id="feedback@ios.doi.gov">
-                  <p>Sally Jewell, Secretary of the Interior</p>
-                </section>
-              </div>
+          </div>
+          <div id="placeholder"></div>
+          <div class= "action step3">
+            <h3>3.  Contact these Representatives</h3>
+            <div class="pics-container">
+              <section class="pol-pics" id="obama">
+                <a href="https://www.whitehouse.gov/contact" target="_blank"><img src="photos/obama.jpg" alt="http://a5.files.biography.com/image/upload/c_fill,cs_srgb,dpr_1.0,g_face,h_300,q_80,w_300/MTE4MDAzNDEwNzg5ODI4MTEw.jpg" title="Barack Obama" class="pol-button" id="president@whitehouse.gov">
```

Comment:
I'm not sure I understand the use of the id here. If you are trying to save the email address on the node, using a data attribute like data-email="president@whitehouse.gov" would be more appropriate. 
If the id is just for selecting, then I'd go with something that's the same pattern for each image like obama-image, inslee-image, cantwell-image.